k8s/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: doks-debug
+  labels:
+    app: doks-debug
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: doks-debug
+  template:
+    metadata:
+      labels:
+        name: doks-debug
+    spec:
+      containers:
+          securityContext:
+            privileged: true
+          image: digitalocean/doks-debug:latest
+          name: doks-debug
+          command: [ "sleep", "infinity" ]
```

Comment:
Should we also include the same volume mounts in the deployment version? I can see how it feels less relevant since we aren't targeting a host, but it might be useful to sanity check that our worker node image bootstrapped correctly.